packages/itwin/grouping-mapping-widget/src/widget/hooks/useValidator.ts

```diff
@@ -15,13 +15,15 @@ export const Validators = {
   OnlyBeginsWithLetterOrUnderscore: {
     message: "Name can only start with a letter or underscore.",
     rule: (val: string) => {
-      return !val.match(/^[\W\d]+/i);
+      var regexPattern = new RegExp(/^[\p{L}\p{Nl}_]+/u)
+      return regexPattern.test(val);
     },
   },
   FollowedByLettersUnderscoresAndDigits: {
     message: "Name can only contain letters, underscores, or digits",
     rule: (val: string) => {
-      return !val.match(/\W+/i);
+      var regexPattern = new RegExp(/[\p{L}\p{Nl}\p{Nd}\p{Mn}\p{Mc}\p{Pc}\p{Cf}]+$/u)
```

Comment:
Do not use `var`. Use `const`.
Also, you are missing a semicolon at the end of this line. Run `rush lint` to check for these errors.